src/cli/src/consume/mod.rs

```diff
@@ -89,7 +89,7 @@ impl ConsumeOpt {
         name = "Consume",
         fields(topic = %self.topic, partition = self.partition),
     )]
-    pub async fn process(self, fluvio: &Fluvio) -> Result<(), ConsumerError> {
+    pub async fn process(self, fluvio: &Fluvio) -> Result<()> {
```

Comment:
reason why this have to be `CliError` ?  It's better to scope to specific Error

Reply:
CliError makes the most sense here. `ConsumerError` was a shim type that was overly specific and was actually mis-applied in a lot of places (e.g. an error creating a topic would be considered a "ConsumerError"). We don't really lose anything by removing this extra layer of nesting.